integr8ly/delorean
# Del 300 product downtime

https://issues.redhat.com/browse/DEL-300

to verify:

1. Prepare  a RHMI cluster and have the workload app installed.
2. Run the following command from this branch:
   ```
   ./delorean pipeline query-report \
      --config-file ./configurations/downtime-report-config.yaml \
      -o /tmp
   ```
3. Once finished, check the output file
